core/src/main/java/edu/wpi/grip/core/operations/network/ros/RealROSManager.java

```diff
@@ -0,0 +1,184 @@
+package edu.wpi.grip.core.operations.network.ros;
+
+
+import com.google.common.collect.ImmutableSet;
+import com.google.common.collect.Maps;
+import com.google.common.collect.Sets;
+import edu.wpi.grip.core.operations.network.NetworkPublisher;
+import org.ros.concurrent.CancellableLoop;
+import org.ros.internal.message.Message;
+import org.ros.namespace.GraphName;
+import org.ros.node.*;
+import org.ros.node.topic.Publisher;
+
+import java.util.*;
+import java.util.concurrent.ConcurrentMap;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+/**
+ *
+ */
+public class RealROSManager implements ROSManager {
```

Comment:
Just make this `ROSManager`.  Don't pollute the code with ridiculous classes for the sake of working around weird testing methods.